test/test_jit.py

```diff
@@ -1534,23 +1533,45 @@ def test_if_while(a, b) -> (c):
         self.checkScript(script, inputs, outputs, False, 'test_if_while')
 
     def test_script_ternary(self):
-        cu = torch.jit._jit_script_compile('''
+        cu = torch.jit.CompilationUnit('''
         def test_ternary_control(a, b) -> (c):
             c = 3
             if a > 3:
                 c = a + b
             else:
                 c = b
         ''')
-        cu2 = torch.jit._jit_script_compile('''
+        cu2 = torch.jit.CompilationUnit('''
         def test_ternary(a, b) -> (c):
             c = 3
             c = a + b if a > 3 else b
         ''')
         self.assertEqual(
-            str(cu.get_graph('test_ternary_control')),
-            str(cu2.get_graph('test_ternary')),
+            str(cu.cu.get_graph('test_ternary_control')),
+            str(cu2.cu.get_graph('test_ternary')),
```

Comment:
Why do we have to index it twice now?

Reply:
The frontend-facing CompilationUnit doesn't expose graphs, just the GraphExecutor objects that run the graphs. I just reached into the members of CompilationUnit here to get to the underlying C struct so we can check graph equality. If in the future we see the need for users to call get_graph, we can create a proxy method.

Reply:
Oh I somehow entirely missed the fact that you added a Python class for it...